### ISSUE TITLE
Editing blogs under rails 4.2 crashes

### ISSUE THREAD

**Issue by Atastor** [refinery/refinerycms-blog#424]
Only now noticed something which I introduced last week. bundle-updating my application introduced a silent rails 4.2 update (via refinerycms-page-images new dependency from globalize 5 via commit 33586a198df0c72adffc828ea89ef4517ca5ba75). Since then when editing a blog I get a crash:

![bildschirmfoto 2015-03-08 um 08 54 43](https://cloud.githubusercontent.com/assets/21538/6545052/df000bee-c570-11e4-9fcb-c35a88e39463.png)

Fixing page-images on the last commit before the above mentioned and thus keeping rails etc. at 4.1.9 fixes the error, so I do think the error is the combination of rails 4.2 and current refinerycms-blog.

Any ideas?

Thanks
Michael

**Comment by maxmiliano** [comment]
I solved this by creating a decorator for `Refinery::Blog::Admin::PostsController` and overriding the `find_post` to this:

    def find_post
      @post = Refinery::Blog::Post.friendly.find(params[:id])
    end

I found this solution after reading a discussion on this forum, but I can't find id anymore. 
It worked for me.

It was a problem on `Refinery::Blog::Post.find_by_slug(params[:id]) if params[:id].friendly_id?`

Or something like that.
